field_history/migrations/0001_initial.py

```diff
@@ -12,7 +12,6 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('contenttypes', '0002_remove_content_type_name'),
```

Comment:
it's for consistency with Django 1.7.5. At that time there was no such migration